src/Components/Input.jsx

```diff
@@ -0,0 +1,51 @@
+/**
+ * InputFiled Component
```

Comment:
Ideally this part should provide a clear description of what the component is. For example it could be something like `This returns an input field of a given type`

Reply:
Fixed 8b7142b3bb568f0c1e361ef10bfeae717284050f